cmd/test-utils_test.go

```diff
@@ -366,6 +367,20 @@ func StartTestStorageRPCServer(t TestErrHandler, instanceType string, diskN int)
 	testRPCServer.AccessKey = credentials.AccessKey
 	testRPCServer.SecretKey = credentials.SecretKey
 
+	// Create dummy format json file.
+	emptyConfig := []byte("{}")
```

Comment:
This is just a workaround to make `verifyServerFunc()` to work.  Even if we have real/working configuration, its no use.